login.go

```diff
@@ -0,0 +1,111 @@
+/*
+   Copyright (C) nerdctl authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli/v2"
+	"io/ioutil"
+	"os"
+	"strings"
+)
+
+type loginOptions struct {
+	serverAddress string
+	username      string
+	password      string
+	passwordStdin bool
+}
+
+func NewLoginCommand() *cli.Command {
```

Comment:
Was this ported from https://github.com/genuinetools/img/blob/v0.5.11/login.go ?
If so, please clarify the origin in the code comment.

Reply:
@AkihiroSuda  fixed